### ISSUE TITLE
Scribe-Data Outreach 👋🚀

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#178]
### Terms

- [X] I have searched [open and closed issues](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

## Sharing and promoting Scribe-Data

**Comment by andrewtavis** [comment]
Ping @mhmohona 👋 Let me know if you have ideas here!

**Comment by to-sta** [comment]
I can post about Scribe-Data in the [ReDI School](https://www.redi-school.org/) Slack channel. It might be beneficial for some of the students to gain experience through contributions.

**Comment by andrewtavis** [comment]
Hey @to-sta! Ya, we'd be very happy to work with ReDI School students! For Scribe-Data, but also if there are students working on Swift for Scribe-iOS or Kotlin for Scribe-Android (we're making lots more progress on Android these days).

Thanks for the offer! 😊

**Comment by LevisNgigi** [comment]
Hey, I am an outreachy applicant and I can share more about Scribe-data on our [Moringa School](https://moringaschool.com/) whatsapp Alumni channel.We currently have over 800 members. Could that help?

**Comment by andrewtavis** [comment]
Could definitely help, @LevisNgigi! There are some updates coming to the software in the coming week, so should we wait on that?

**Comment by DeleMike** [comment]
Hi @andrewtavis, nice meeting you! I am an outreachy applicant and a Nigerian, I was wondering if this app could help with Yoruba language learning, especially data-wise. Could it help to structure the language data in a way that is usable in the app? I know a lecturer that is working on developing a Yoruba language application for kids and one of their current challenges is data structure and content. is there a way Scribe-Data can help?

Here is a highlight about [KoEde on LinkedIn](https://www.linkedin.com/posts/aderonke-sakpere-678b8a62_hlff-spotlight-alumni-in-action-heidelberg-activity-7062054002426822658-hiud?utm_source=share&utm_medium=member_desktop)

**Comment by LevisNgigi** [comment]
> Could definitely help, @LevisNgigi! There are some updates coming to the software in the coming week, so should we wait on that?

That is definitely okay.Will be looking forward to it.

**Comment by andrewtavis** [comment]
> Hi @andrewtavis, nice meeting you! I am an outreachy applicant and a Nigerian, I was wondering if Scribe-Data could help with Yoruba language learning, especially data-wise. Could it help to structure the language data in a way that is usable in the app? I know a lecturer that is working on developing a Yoruba language application for kids and one of their current challenges is data structure and content. is there a way Scribe-Data can help?
> 
> Here is a highlight about [KoEde on LinkedIn](https://www.linkedin.com/posts/aderonke-sakpere-678b8a62_hlff-spotlight-alumni-in-action-heidelberg-activity-7062054002426822658-hiud?utm_source=share&utm_medium=member_desktop)

Hey @DeleMike 👋 Yoruba isn't a currently supported language, but could be 🤔 Would you want me to make an issue for adding as much as we can and you could work on that? :)

**Comment by LevisNgigi** [comment]
Hi @andrewtavis I was having a look at [language directories in src/scribe_data/language_data_extraction](https://github.com/scribe-org/Scribe-Data/tree/main/src/scribe_data/language_data_extraction) and I did not find Swahili which is my native language.It is a language that is spoken mainly around the East African region and some other parts of Africa.If possible I would love to work on it?

**Comment by andrewtavis** [comment]
Would be great, @LevisNgigi! I'll make this issue now :)

**Comment by DeleMike** [comment]
> > Hi @andrewtavis, nice meeting you! I am an outreachy applicant and a Nigerian, I was wondering if Scribe-Data could help with Yoruba language learning, especially data-wise. Could it help to structure the language data in a way that is usable in the app? I know a lecturer that is working on developing a Yoruba language application for kids and one of their current challenges is data structure and content. is there a way Scribe-Data can help?
> > Here is a highlight about [KoEde on LinkedIn](https://www.linkedin.com/posts/aderonke-sakpere-678b8a62_hlff-spotlight-alumni-in-action-heidelberg-activity-7062054002426822658-hiud?utm_source=share&utm_medium=member_desktop)
> 
> Hey @DeleMike 👋 Yoruba isn't a currently supported language, but could be 🤔 Would you want me to make an issue for adding as much as we can and you could work on that? :)

Yes, I would love to work on it! Please create the issue, thanks. We would discuss in detail about this in sync soon. Thanks!

**Comment by andrewtavis** [comment]
@LevisNgigi, would you want to make the issue for Yaruba? I need to step out :) You can make it similar to the Swahili one.

**Comment by LevisNgigi** [comment]
@andrewtavis , Yes I could make the issue. Doing it now

**Comment by andrewtavis** [comment]
Ah sorry I'm on my phone and I pinged @LevisNgigi and meant to ping @DeleMike :) Either or would be great, and let's then let @DeleMike know about it and then I can assign 😊

**Comment by LevisNgigi** [comment]
I have opened the issue @DeleMike . Also I think I do not have permission to add more labels and to include the scribe board under projects.I think @andrewtavis will help with that.

**Comment by andrewtavis** [comment]
Thanks for the help opening the issue, @LevisNgigi!

**Comment by DeleMike** [comment]
Thanks @LevisNgigi! 🙌🏾

**Comment by LevisNgigi** [comment]
Pleasure is mine.

**Comment by swanjikumainaa** [comment]
Hello,
@andrewtavis this is great stuff.
I’m Susan, a software engineer very passionate about tech events and communities. I love the ideas about promoting Scribe-Data. I'm currently getting started with creating tech content on platforms like Instagram, TikTok, and Twitter, and I've seen these socials going a long way in reaching more users and generally educating people. I also think, partnering with tech events and offering swag to students at events we partner with could create a movement or community and encourage participation.

What do you think about this?

**Comment by andrewtavis** [comment]
Hi @swanjikumainaa 👋 These are interesting ideas :) I have to say that I'm not a social media user, so I wouldn't know where to begin on these things and likely wouldn't be too involved, but if others from the community would like to look into this it'd be ok on my end. As far as giveaways as well, not really sure what we could do there as we don't have anything like this :)

**Comment by GicharuElvis** [comment]
Hi, anyone working on this?

**Comment by andrewtavis** [comment]
Thanks for sharing the project with your communities, @keerthanakh89!

**Comment by pranavs321** [comment]
Hi! I'm Pranav, a CS student from India interested in Python and open source. I found Scribe through the Wikimedia good first tasks list. I'd love to contribute and learn from this community!

**Comment by andrewtavis** [comment]
Great to have you here, @pranavs321! You'd be welcome to work on Scribe-Data and other Scribe projects! If you want to learn a bit of Android coding, then [Scribe-Android](https://github.com/scribe-org/Scribe-Android) is actively moving towards an initial release. As far as Scribe-Data is concerned, we have a few issues that could be worked on, namely #623 and #703 (the second one is blocked for now). If you'd like to write some tests for the project, then by all means write into #623 and we'll assign you 😊
